duckduckgo/iOS
# Redesign "Use Duck Duck Go in Safari" screen

Task/Issue URL: https://app.asana.com/0/361428290920652/689645049709948
Tech Design URL: N/A

**Description**:
* Update "Use DuckDuckGo in Safari" screen to better match new modal popup design
* Remove disclosure indicators from modal popups in settings
* Switch modals popups to cornflower blue 

**Steps to test this PR**:
1. Go to Settings
1. Open "Use DuckDuckGo in Safari" and ensure it renders ok
1. Switch from portrait to landscape mode and again ensure it renders fine
1. Press the "Done" button to close it
1. Open "Use DuckDuckGo in Safari" again
1. Press the 'x' button to close it
Repeat on a few different devices

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
